docs/common-api/objects.rst

```diff
@@ -218,7 +218,7 @@ Execution Result Object
    * - ``status``
      - ``enum[str]``
 
-     - One of ``"continued"``, ``"waiting-input"``, ``"finished"``, or ``"build-finished"``.
+     - One of ``"continued"``, ``"waiting-input"``, ``"finished"``, ``"build-finished"``, or ``"exec-timeout"``.
        See more details at :ref:`code-execution-model`.
```

Comment:
Shouldn't we add `clean-finished` which you have added for lablup/backend.ai-agent#74 ?

Reply:
I thought that should go later, not this PR, because this PR is for 1.4 branch...

Reply:
...and I found this PR merges into master! I'll add `clean-finished` :D